pkg/kube/snapshot/types.go

```diff
@@ -0,0 +1,13 @@
+package snapshot
+
+// ObjectMeta is metadata of snapshot which is configurable.
```

Comment:
it's not just the metadata of the snapshot right? It can basically be used for anything.